src/Javis.jl

```diff
@@ -561,6 +563,17 @@ Returens the final rendered frame
 """
 function get_javis_frame(video, objects, frame; layers = Layer[])
 
+    #extract object jpaths on first frame
+    if frame == 1
+        for object in objects
+            try
+                println("EXTRACTING " * object.opts[:name] * " JPATHS")
```

Comment:
I'm confused by the `try-catch` block here.

Reply:
this will be removed , just a hack around to  debug  code.